PlaygroundService/Controllers/PlaygroundController.cs

```diff
@@ -22,6 +23,25 @@ public PlaygroundController(IClusterClient client, ILogger<PlaygroundController>
             _client = client;
             _logger = logger;
         }
+        
+        [HttpGet("templates")]
+        public async Task<IActionResult> GetTemplateConfig()
+        {
+            _logger.LogInformation("templates  - GetTemplateConfig started time: "+ DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss"));
+            var templateConfGrain = _client.GetGrain<IPlaygroundGrain>("userId");
+            var templateConf = await templateConfGrain.GetTemplateConfig();
+
+            return Ok(templateConf); 
+        }
+        
+        [HttpGet("templateInfo")]
+        public async Task<IActionResult> GetTemplateInfo([FromQuery] string template, [FromQuery] string projectName)
+        {
+            _logger.LogInformation("templates  - GetTemplateInfo started time: "+ DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss"));
+            var codeGeneratorGrain = _client.GetGrain<IPlaygroundGrain>("userId");
```

Comment:
Same here for userId, this will cause the same grain to be reused, ignoring the flexibility feature of Orleans.